appcues/src/main/java/com/appcues/data/remote/appcues/AppcuesRemoteSource.kt

```diff
@@ -43,8 +41,8 @@ internal class AppcuesRemoteSource(
         experienceId: String,
         userSignature: String?,
     ): ResultOf<ExperienceResponse, RemoteError> =
-        // preview _can_ be personalized, so attempt to use the user info, if a valid session exists
-        if (sessionMonitor.isActive) {
+        // preview _can_ be personalized, so attempt to use the user info, if a valid userId exists
+        if (storage.userId.isNotEmpty()) {
```

Comment:
this is just a simplification and dependency removal - we can do a personalized preview as long as we have the userId